beacon-chain/core/helpers/committee.go

```diff
@@ -31,6 +32,39 @@ type shufflingInput struct {
 	committeesPerEpoch uint64
 }
 
+// ValidatorAssignment defines the assignment information needed by
+// validators every epoch.
+type ValidatorAssignment struct {
+	Committee  []uint64
+	Shard      uint64
+	Slot       uint64
+	IsProposer bool
+}
+
+var currentEpoch uint64
+var validatorAssignments map[uint64]*ValidatorAssignment
+var assignmentsLock sync.RWMutex
+
+// ClearAssignmentsCache clears the assignments stored to make
+// sure the assignments data is recalculated.
+func ClearAssignmentsCache() {
```

Comment:
why have this if it is never used ?

Reply:
Good catch, the later changes I made removed it's need.